python/ray/data/dataset.py

```diff
@@ -507,8 +507,6 @@ def random_shuffle(
         *,
         seed: Optional[int] = None,
         num_blocks: Optional[int] = None,
-        _spread_resource_prefix: Optional[str] = None,
```

Comment:
@scv119 Have we confirmed with Uber that they no longer need `_spread_resource_prefix`?

Reply:
AFAIK we have not.